### PR TITLE
Add case for "My issues" on index page

### PR DESCRIPTION
**Pull request by huiwangredhat**
[OSIDB-2384](https://issues.redhat.com/browse/OSIDB-2384)

### CODE REVIEW

File: features/steps/flaw_filter.py [hunk 1]
```diff
@@ -100,3 +100,19 @@ def step_impl(context):
...
+    home_page = HomePage(context.browser)
+    home_page.flaw_list_exist()
+    home_page.click_btn("myissueCheckBox")
+
+@then("All issues assigned to me should be listed in flaw table")
+def step_impl(context):
+    home_page = HomePage(context.browser)
+    user_name = home_page.userBtn.get_text()
+    home_page.flaw_list_exist()
+    owner = home_page.get_field_value("owner")
```
[the remark this answers is not in the document]
Comment (huiwangredhat) [reply features/steps/flaw_filter.py:120]: Updated.

[the remark this answers is not in the document]
Comment (huiwangredhat) [reply features/steps/flaw_filter.py:137]: Updated again due to the bulk assign removed from the UI. pls help to review it again. Thanks

**Review by RichardXuan — APPROVED** [review]
LGTM

File: features/pages/advanced_search_page.py [hunk 2]
```diff
@@ -4,6 +4,9 @@
 from seleniumpagefactory.Pagefactory import PageFactory
 from selenium.webdriver.common.by import By
 from features.constants import OSIDB_URL
+from features.utils import (
+        go_to_advanced_search_page
+)
```
[the remark this answers is not in the document]
Comment (huiwangredhat) [reply features/pages/advanced_search_page.py:9]: Ok, I will update it in another PR.

**Pull request merged**